cmd/utils.go

```diff
@@ -542,38 +542,21 @@ func GenKnativeYaml(yamlTemplate string, deployPort int, serviceName string, dep
 		return "", err
```

Comment:
FUTURE: I think in future refactoring we might want to consider moving the k8s functions out of utils.go into a k8utils.go etc, like we are planning on doing for docker commands.